### PR TITLE
styling done yitik

### PR DESCRIPTION
**Pull request by yitik-kawatra**
challenge complete #50

### CODE REVIEW

File: participants/yitik-kawatra.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
-Name: Yitik Kawatra
-GitHub username: yitik-kawatra
-University: GGSIPU
-Year: 2nd
-Branch: ECE
+<h1>Name: Yitik Kawatra</h1>
+<h2>GitHub username:[yitik-kawatra](https://https://github.com/yitik-kawatra/
```
Comment (soumyaa1804) [inline participants/yitik-kawatra.md:2]: You forgot a `)` at the end of your GitHub link.

File: participants/yitik-kawatra.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+</h2>
+<div>
+<h3>
+<i>
+<ul>
+<li>*University: GGSIPU </li>
```
Comment (lisa761) [inline participants/yitik-kawatra.md:8]: Perhaps the asterisk here was a mistake? You can view your file [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/bc1cc28f55127610883b74824479899efdabb1f0/participants/yitik-kawatra.md) to see what I am talking about.

File: participants/yitik-kawatra.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
-Name: Yitik Kawatra
-GitHub username: yitik-kawatra
-University: GGSIPU
-Year: 2nd
-Branch: ECE
+<h1>Name: Yitik Kawatra</h1>
+<h2>GitHub username:[yitik-kawatra](https://https://github.com/yitik-kawatra/
+</h2>
+<div>
+<h3>
```
Comment (lisa761) [inline participants/yitik-kawatra.md:5]: Don't you think having an entire list inside an `<h3>` tag makes the list bold for no reason?

File: participants/yitik-kawatra.md (hunk 1 above)
Comment (lisa761) [inline participants/yitik-kawatra.md:2]: You have forgotten the closing `)` for the link here.

File: participants/yitik-kawatra.md (hunk 2 above)
Comment (yitik-kawatra) [reply participants/yitik-kawatra.md:8]: @lisa761 Thank you I will make the necessary changes and submit again.

**Comment by lisa761** [comment]
Great! So if you're done, tell us, so we can merge this

**Comment by yitik-kawatra** [comment]
> Great! So if you're done, tell us, so we can merge this

@lisa761  done

**Pull request merged**